spec/rest_in_peace/template_sanitizer_spec.rb

```diff
@@ -120,12 +120,20 @@
 
       context 'param which need to be encoded' do
         let(:params) { { id: 'this param need to be encoded' } }
-        let(:encoded_param) { 'this+param+need+to+be+encoded' }
+        let(:encoded_param) { 'this%20param%20need%20to%20be%20encoded' }
```

Comment:
Hello @cimnine 
I did not changed the `input` of the test. I changed the output of the test.
Previously space separated params was expected to be parsed as `+` separated string(`params to encode` => `params+to+encode`)
After my fix, space separated params is expected to be parsed as `%20` separated string(`params to encode` => `params%20to%20encode`). The original case should not still work. 😄